gnomad_mitochondria/pipeline/add_annotations.py

```diff
@@ -142,13 +144,49 @@ def add_gnomad_metadata(input_mt: hl.MatrixTable) -> hl.MatrixTable:
     return input_mt
 
 
-def filter_by_copy_number(input_mt: hl.MatrixTable) -> hl.MatrixTable:
+def add_age_and_pop(input_mt: hl.MatrixTable, all_output: str) -> hl.MatrixTable:
```

Comment:
there's another function that uses "all_output", which is the name of a main input arg,  so i'm wondering if it would be more confusing to rename it just for this function? I might actually rename this everywhere to metadata_tsv